hcam_finder/hcam_finder.py

```diff
@@ -77,7 +77,10 @@ def saveconf(self):
         # target info
         target = dict()
         target["target"] = self.targName.value()
-        targ_coord = SkyCoord(self.targCoords.value(), unit=(u.hour, u.deg))
+        if self.have_decimal_coords():
+            targ_coord = SkyCoord(self.targCoords.value(), unit=(u.deg, u.deg))
+        else:
+            targ_coord = SkyCoord(self.targCoords.value(), unit=(u.hourangle, u.deg))
```

Comment:
minor fix, but this should be a unit of `u.hour`, not `u.hourangle`. They transform 1:1 but they're not the same thing...

Reply:
Could I ask what the difference is? All of the astropy SkyCoords examples use u.hourangle, not u.hour so I assumed u.hourangle was the correct one (and have been using that in all my own code)

Reply:
OK - feel free to keep it in `hourangle`. The difference is in the zeropoint. Hours of RA start with 0 at north, hour angle has 0 to the south

Reply:
I'm happy to change it of course if you prefer u.hour, I was just wondering if I'd been using the wrong unit in all my code for years and didn't realise there was a difference between them